interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a system that tracks changes in the computer files Git hub is the place where the changes are logged and reviewed, it allows for collaborative work. Github is similar to the cloud whereas the changes made on your computer dont effect the files on github but you can use the terminal to push updates through.
 
-Researched answer:
+Researched answer: Git is a version control system used for tracking changes in computer files. It is generally used for source code management in software development. GitHub is an increasingly popular programming resource used for code sharing.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The equality, relational, and logical operators will return a boolean value.
 
-Researched answer:
+Researched answer: Equality operators compare one value to the other and judge if the values equate to the same thing returning a boolean value. Relational operators looks for a relationship between two values. A Boolean value is returned. Logical operators also return a boolean value.
```

Comment:
Like where you were heading with the research answer. You started solid with the equality operators. Recommend adding more description to the relational and logical operators.